aws/resource_aws_emr_instance_group.go

```diff
@@ -177,6 +177,12 @@ func resourceAwsEMRInstanceGroupRead(d *schema.ResourceData, meta interface{}) e
 		return fmt.Errorf("error reading EMR Instance Group (%s): %s", d.Id(), err)
 	}
 
+	if *ig.Status.State == "TERMINATED" {
```

Comment:
To prevent potential panics, can you please add `nil` checking here and use some of the AWS Go SDK goodies? 😄 

```suggestion
	if ig.Status != nil && aws.StringValue(ig.Status.State) == emr.InstanceGroupStateTerminated {
```

Reply:
Sigh, all these pointers are really annoying. Though, I suppose, this does leave the question of how to treat `ig.Status` being `nil` -- is it an error condition? Does it mean the IG doesn't exist? Something else?